client/scss/components/_help-block.scss

```
@@ -60,25 +60,16 @@
 }
 
 // Media for Windows High Contrast
-@media (forced-colors: $media-forced-colours) {
+@media (forced-colors: active) {
   .help-block {
-    forced-color-adjust: none;
-    border: 1px solid $system-color-link-text; // ensure visible separation in Windows High Contrast mode
-    background-color: transparent;
-    color: $color-white;
-
-    &:before {
-      color: currentColor;
-    }
+    border: 3px solid $system-color-link-text; // ensure visible separation in Windows High Contrast mode
   }
 
   .help-warning {
-    color: $color-text-warning-forced-color;
-    border-color: $color-text-warning-forced-color;
+    border: 3px dotted $color-text-warning-forced-color;
```

`$color-text-warning-forced-color` isn’t doing anything without `forced-color-adjust: none`, so this could be replaced by:

```suggestion
    border-style: dotted;
```

All other border styles will inherit from `help-block` above.